testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/KeyVaultAuthHelper.java

```diff
@@ -49,10 +50,28 @@
     private final static String MSSTS_CLIENT_ASSERTION_AUDIENCE = "https://login.microsoftonline.com/microsoft.com/oauth2/v2.0/token";
 
     private static KeyVaultAuthHelper sKeyVaultAuthHelper;
+    private final String mSecret;
+
+    private KeyVaultAuthHelper(final String secret) {
+        mSecret = secret;
+    }
```

Comment:
Is this constructor being used anywhere?

Reply:
In the static getInstanceWithSecret factory.